src/main/kotlin/pt/up/fe/ni/website/backend/model/Event.kt

```diff
@@ -1,15 +1,26 @@
 package pt.up.fe.ni.website.backend.model
 
+import com.fasterxml.jackson.annotation.JsonProperty
 import java.util.Date
 import javax.persistence.Entity
 import javax.persistence.GeneratedValue
 import javax.persistence.Id
+import javax.validation.constraints.Size
+import pt.up.fe.ni.website.backend.model.constants.EventConstants as constants
 
 @Entity
 class Event(
+    @JsonProperty(required = true)
```

Comment:
To clarify, I'm not saying they aren't needed. I'm just curious about the reason so I can be more careful in my own PRs.

Reply:
This annotation makes no difference here. However, I was playing with numerical fields and the this annotation is needed in those, otherwise Jackson uses the default value instead of throwing an error. I assume this happens for every class that is mapped to a primitive in Java. I decided to put the annotations everywhere for consistency

Reply:
Thanks!